src/TabbedBehavior.js

```diff
@@ -83,6 +83,8 @@ define(function (require) {
             tabView.$el.addClass('selected');
             var newView = new (tab.get('ContentView'))(options);
             this.view.content.show(newView);
+
+            this.view.trigger('marionette:tabs:switch', {tab: tab});
```

Comment:
It's a sort of namespace, reserved for our plugin. Just to not intersect with users events (unlikely, but still possible).